<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:green;border:1px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
